packages/manifold/contracts/edition/ManifoldERC721Edition.sol

```diff
@@ -47,83 +48,95 @@ contract ManifoldERC721Edition is CreatorExtension, ICreatorExtensionTokenURI, I
+    function setTokenURIPrefix(address creatorCore, uint256 instanceId, string calldata prefix) external override creatorAdminRequired(creatorCore) {
+        require(_maxSupply[creatorCore][instanceId] != 0, "Invalid instanceId");
+        _tokenPrefix[creatorCore][instanceId] = prefix;
     }
     
     /**
      * @dev See {ICreatorExtensionTokenURI-tokenURI}.
      */
-    function tokenURI(address creator, uint256 tokenId) external view override returns (string memory) {
-        (uint256 series, uint256 index) = _tokenSeriesAndIndex(creator, tokenId);
-        return string(abi.encodePacked(_tokenPrefix[creator][series], (index+1).toString()));
+    function tokenURI(address creatorCore, uint256 tokenId) external view override returns (string memory) {
+        (uint256 instanceId, uint256 index) = _tokenInstanceAndIndex(creatorCore, tokenId);
+        return string(abi.encodePacked(_tokenPrefix[creatorCore][instanceId], (index+1).toString()));
     }
     
     /**
      * @dev See {IManifoldERC721Edition-mint}.
      */
-    function mint(address creator, uint256 series, address recipient, uint16 count) external override nonReentrant creatorAdminRequired(creator) {
+    function mint(address creatorCore, uint256 instanceId, address recipient, uint16 count) external override nonReentrant creatorAdminRequired(creatorCore) {
         require(count > 0, "Invalid amount requested");
-        require(_totalSupply[creator][series]+count <= _maxSupply[creator][series], "Too many requested");
+        require(_totalSupply[creatorCore][instanceId]+count <= _maxSupply[creatorCore][instanceId], "Too many requested");
         
-        uint256[] memory tokenIds = IERC721CreatorCore(creator).mintExtensionBatch(recipient, count);
-        _updateIndexRanges(creator, series, tokenIds[0], count);
+        uint256[] memory tokenIds = IERC721CreatorCore(creatorCore).mintExtensionBatch(recipient, count);
+        _updateIndexRanges(creatorCore, instanceId, tokenIds[0], count);
     }
 
     /**
      * @dev See {IManifoldERC721Edition-mint}.
      */
-    function mint(address creator, uint256 series, address[] calldata recipients) external override nonReentrant creatorAdminRequired(creator) {
+    function mint(address creatorCore, uint256 instanceId, address[] calldata recipients) external override nonReentrant creatorAdminRequired(creatorCore) {
         require(recipients.length > 0, "Invalid amount requested");
-        require(_totalSupply[creator][series]+recipients.length <= _maxSupply[creator][series], "Too many requested");
+        require(_totalSupply[creatorCore][instanceId]+recipients.length <= _maxSupply[creatorCore][instanceId], "Too many requested");
         
-        uint256 startIndex = IERC721CreatorCore(creator).mintExtension(recipients[0]);
+        mintTokens(creatorCore, recipients, instanceId);
+    }
+
+    function mintTokens(address creatorCore, address[] memory recipients, uint256 instanceId) internal {
```

Comment:
Naming.  Change to _mintTokens (usually internal or private should have underscore in front).

Why not private? we don't intend to inherit this right?

Reply:
done